sympy/geometry/parabola.py

```diff
@@ -0,0 +1,352 @@
+"""Parabolic geometrical entity.
+
+Contains
+* Parabola
+
+"""
+
+from __future__ import division, print_function
+
+from sympy.core import S
+from sympy.core.numbers import oo
+
+from sympy.geometry.entity import GeometryEntity, GeometrySet
+from sympy.geometry.point import Point
+from sympy.geometry.line import Line
+from sympy.geometry.util import _symbol
+
+
+class Parabola(GeometrySet):
+    """A parabolic GeometryEntity.
+
+    A parabola is declared with a point, that is called 'focus', and
```

Comment:
Thank you for your feedback. Yes we had thought about that but we didn't have the time to dive deep in it. We focused on some basic stuff and implementing them right but we definitely intend to add more functionalities or options to the already existing ones once we have more time in our hands.